indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -559,9 +559,11 @@ def _process(self):
                          .all())
         signals.event.designer.print_badge_template.send(self.template, regform=self.regform,
                                                          registrations=registrations)
+        ticket_badges = config_params.pop('ticket_badges', False)
+        file_prefix = _('Tickets') if ticket_badges else _('Badges')
```

Comment:
I'd prefer to not have i18n here. Translating filenames adds all kinds of weird characters, that will get stripped/replaced when converting to a secure filename (which is not done here btw, so only whatever send_file itself rejects will be removed!) ...

Reply:
okay, I've removed the translation